src/sparse/KokkosSparse_spadd.hpp

```diff
@@ -222,11 +221,7 @@ void spadd_symbolic(KernelHandle* handle, const AMatrix& A, const BMatrix& B,
   row_map_type row_mapC(
       Kokkos::view_alloc(Kokkos::WithoutInitializing, "row map"),
       A.numRows() + 1);
-  KokkosSparse::Experimental::spadd_symbolic<
-      KernelHandle, typename AMatrix::row_map_type::const_type,
-      typename AMatrix::index_type::const_type,
-      typename BMatrix::row_map_type::const_type,
-      typename BMatrix::index_type::const_type, row_map_type, entries_type>(
+  KokkosSparse::Experimental::spadd_symbolic(
```

Comment:
Some unrelated cleanup. Unused template argument clone_nnz_view_t made it so template arguments could not be inferred.